tests/plugins/test_input_osv3.py

```diff
@@ -340,8 +341,14 @@ class TestOSv3SourceContainerInputPlugin(object):
 
     @property
     def user_params(self):
+        osbs_loader = pkgutil.find_loader('osbs')
+        osbs_pkg_file = osbs_loader.get_filename()
+        osbs_pkg_path = os.path.dirname(osbs_pkg_file)
```

Comment:
Is `osbs_pkg_path = os.path.dirname(osbs_loader.path)` what we want here?

Reply:
ImpLoader does not have a `path` attribute in python 2.7

Reply:
Ah!